lib/modules/datasource/go/base.spec.ts

```diff
@@ -394,6 +395,49 @@ describe('modules/datasource/go/base', () => {
 
         expect(res).toBeNull();
       });
+
+      it('it correctly splits a URL where the endpoint is contained', async () => {
+        hostRules.hostType.mockReturnValue('gitlab');
+
+        GlobalConfig.set({ endpoint: 'https://example.com/gitlab/api/v4/' });
```

Comment:
please call global config reset in before each callback

Reply:
Done